src/ch.usi.inf.nodeprof/src/ch/usi/inf/nodeprof/analysis/ProfilerExecutionEventNode.java

```diff
@@ -87,7 +87,7 @@ protected void onInputValue(VirtualFrame frame, EventContext inputContext,
 
                 // allow for handler changes after executePre/Post
                 checkHandlerChanges();
-            } catch (UserScriptException e) {
+            } catch (GraalJSException e) {
```

Comment:
These `catch` clauses are no longer needed. I actually also had a patch to simplify error handling a bit more. Pushed it here as 81342eb.